indico/modules/events/clone.py

```diff
@@ -114,11 +120,20 @@ class EventProtectionCloner(EventCloner):
     is_default = True
     uses = {'event_roles', 'registration_forms'}
 
-    def has_conflicts(self, target_event):
+    def get_conflicts(self, target_event):
+        conflicts = []
+
         if target_event.access_key != '':
-            return True
+            conflicts.append(_('The target event already contains an access key'))
```

Comment:
I think "has" sounds better than "contains" in most of these cases.

Reply:
Replaced `contains` with `has` in all conflict messages